Source/Cogworks.Umbraco.Essentials/Cogworks.Umbraco.Essentials.nuspec

```diff
@@ -23,5 +23,6 @@
   </metadata>
   <files>
     <file src="bin\Release\**\Cogworks.Umbraco.Essentials.dll" target="lib/net472" />
+    <file src="ViewFiles\_ResponsiveImage.cshtml" target="content\ViewFiles\_ResponsiveImage.cshtml" />
```

Comment:
You can prepare where the view will be placed after package install.

Something like:
```
<file src="ViewFiles\_ResponsiveImage.cshtml" target="content\Views\Partials\Shared\_ResponsiveImage.cshtml" />
```

You can try it locally with option to build nuget package from nuspec (perform release build first) and then:
```powershell
nuget pack Cogworks.Umbraco.Essentials.nuspec \
  -Version 1.2.3 \
  -Properties Configuration=Release \
  -OutputDirectory "$PWD\output"
```

Another approach to investigate could be:
 * https://www.ianmariano.com/2013/06/11/embedded-razor-views-in-mvc-4/
 * https://www.titanwolf.org/Network/q/6f4d1e34-5719-4570-9acc-6ecf24fab2ff/y
 * https://samiprogramming.medium.com/how-to-include-files-within-a-nuget-package-272d39323d7e
 * https://github.com/dotnet/aspnetcore/issues/1490
 * https://candordeveloper.com/2013/02/11/nuget-package-mvc-controller-from-live-application/

But the simply way is to add destination to nuspec file.

Reply:
I tried this, this will create the file in the projects where you install the package. so this was the "nicer" way of doing it.

Reply:
So we should investigate this approach with assign additional Razor views reader that can read from different location.

Reply:
@aochmann we need to create Cogworks.Umbraco.Essentials.UI as a new package to include controllers and views basically, but there is no reason to hold this PR as with it is easier than before to use this builder. The next steps (new package creation) as the things that we need to discuss with you 👍